packages/builder/builder-shared/src/types/config/output.ts

```diff
@@ -273,6 +273,10 @@ export interface SharedOutputConfig {
    * Whether to transform SVGs into React components. If true, will treat all .svg files as assets.
    */
   disableSvgr?: boolean;
+  /**
+   * Whether to enable the friendly errors plugin.
+   */
+  disablePrettyErrors?: boolean;
```

Comment:
Why do we need this config

Reply:
We should not add a new config just because the pretty formatter is unstable. Every config needs to be designed and has value.